Examples/Algorithms/TrackFinding/src/TrackFindingAlgorithm.cpp

```diff
@@ -214,6 +262,14 @@ TrackFindingAlgorithm::TrackFindingAlgorithm(Config config,
         "required for staying on seed.");
   }
 
+  if (m_cfg.trackSelectorCfg.has_value()) {
+    m_trackSelector = std::visit(
+        [](const auto& cfg) -> std::optional<Acts::TrackSelector> {
+          return {cfg};
+        },
+        m_cfg.trackSelectorCfg.value());
```

Comment:
In fact, do we even need the lambda any more. Would this work?
```suggestion
    m_trackSelector.emplace(m_cfg.trackSelectorCfg.value());
```